### ISSUE TITLE
Assertion failures in Xcelium don't cause non-zero exit code

### ISSUE THREAD

**Issue by ciglass** [svunit/svunit#165]
Hi, I'm running svunit tests with Xcelium. I noticed that RTL assertion failures do not cause svunit to mark the test as failure or exit with a non-zero exit code. Have you seen similar behavior before? Is there an additional flag or setting that would enable this extra checking?

Thanks!

**Comment by tudortimi** [comment]
I don't think this is easily possible to implement. If would be very tool specific to capture the fact that an assertion fired and to integrate this fact into the test result.

**Comment by GordonMcGregor** [comment]
Isn't this all because the exit code from the simulator command isn't returned ?  Ideally you'd capture the result of the system call that launches the simulator and pass that exit code along, rather than exiting with the status of the user-nag message system call.

**Comment by tudortimi** [comment]
I'm guessing `xrun` exits with a non-zero exit code on assertion failure (or that it's at least possible to configure it to do this). With #257, `runSVUnit` will exit with an internal execution error code when `xrun` fails. Does this fit to your expectations?
